### ISSUE TITLE
Uncaught error in chrome? Could not find module './corejs' from node_modules/promise

### ISSUE THREAD

**Issue by SnappyCroissant** [monounity/karma-typescript#49]
```
16 12 2016 16:25:47.374:INFO [compiler.karma-typescript]: Compiling project using Typescript 2.1.4
16 12 2016 16:25:50.431:INFO [compiler.karma-typescript]: Compiled 6 files in 2478 ms.
16 12 2016 16:25:58.189:INFO [builtin-bundler.karma-typescript]: Bundled imports for 4 file(s) in 7257 ms.
16 12 2016 16:25:58.347:WARN [karma]: No captured browser, open http://localhost:9876/
16 12 2016 16:25:58.360:INFO [karma]: Karma v1.3.0 server started at http://localhost:9876/
16 12 2016 16:25:58.361:INFO [launcher]: Launching browser Chrome with unlimited concurrency
16 12 2016 16:25:58.446:INFO [launcher]: Starting browser Chrome
16 12 2016 16:26:12.696:INFO [Chrome 55.0.2883 (Windows 7 0.0.0)]: Connected on socket /#qu2m2-bCZDPe8uQeAAAA with id 47623928
Chrome 55.0.2883 (Windows 7 0.0.0) ERROR
  Uncaught Error: Could not find module

  './core.js' from

  'C:/Users/TimothyH/Documents/GitHub/relay-neo4j-crud/node_modules/promise/lib' using paths

  [./core.js]
  [C:/Users/TimothyH/Documents/GitHub/relay-neo4j-crud/node_modules/promise/core.js]
  [C:/Users/TimothyH/Documents/GitHub/relay-neo4j-crud/node_modules/promise/core.js.js]
  [C:/Users/TimothyH/Documents/GitHub/relay-neo4j-crud/node_modules/promise/core.jsindex.js]
  [C:/Users/TimothyH/Documents/GitHub/relay-neo4j-crud/node_modules/promise/core.js/index.js]

  at node_modules/karma-typescript/lib/bundlers/builtin/commonjs.js:48
```

Getting the same error replicated across two machines one Windows 7 one MacOS Sierra.

Pretty new to typescript and karma so I'm flying a little blind here any help is much appreciated!

![capture](https://cloud.githubusercontent.com/assets/1546076/21252985/327662f8-c3af-11e6-9b43-b85c664123ec.PNG)
The file is definitely there. And in the lib path as well.

UPDATE:
By manually requiring all the plugins in karma conf rather than implicitly and then including core.js manually in the files configuration option I've developed the error to:

Uncaught ReferenceError: require is not defined
at node_modules/promise/lib/core.js:3

So it looks like maybe common.js isn't properly bundling for the browser?

**Comment by SnappyCroissant** [comment]
![capture](https://cloud.githubusercontent.com/assets/1546076/21252985/327662f8-c3af-11e6-9b43-b85c664123ec.PNG)
The file is definitely there.

**Comment by erikbarke** [comment]
Hey @SnappyCroissant, this error was fixed in #38 and the fix will be released in 2.1.5 this weekend!

**Comment by SnappyCroissant** [comment]
Awesome!
